dffml/feature/feature.py

```diff
@@ -75,9 +75,9 @@ class Data(Task):
 
     LOGGER = LOGGER.getChild("Data")
 
-    def __init__(self, src_url: str) -> None:
-        super().__init__(key=src_url)
-        self.src_url = src_url
+    def __init__(self, key: str) -> None:
+        super().__init__(_key=key)
+        self.key = key
```

Comment:
Probably name should not be `_key`.

Reply:
This doesn't really matter, it'll be removed via #303 and #289